docs/swagger.yml

```diff
@@ -0,0 +1,159 @@
+openapi: 3.0.0
+info:
+  description: "GLADoS Auditing API."
+  version: "1.0.0"
+  title: "GLADoS API"
+  
+tags:
+- name: "audit"
+  description: "Provides the ability to store and request audit logs"
+    
+components:
+  securitySchemes:
+    bearerAuth:
+      type: http
+      scheme : bearer
+
+  schemas:
+    new_audit_entry:
+      properties:
+        content:
+          type: string
+          description: "Audit entry message"
+      
+        serviceName:
+          type: string
+          description: "The name of the service"
+    
+        timestamp:
+          type: string
+          description: "ISO8601 without local timestamp"
+          
+        userId:
+          type: string
+          description: "The user's unique ID"
+      required:
+        - content
+        - serviceName
+        - timestamp
+        - userId
+        
+    audit_entry:
+      allOf:
+        - $ref: '#/components/schemas/new_audit_entry'
+      properties:
+        logId:
+          type: string
+          description: "The log's UUID"
+      required:
+        - logId
+
+# Apply security globally  
+security:
+  - bearerAuth: []
+      
+    
+paths: 
+  /audit/{logId} :
+    get:
+      tags: [audit]
+      summary: Gets all log entries matching the given log ID
```

Comment:
Perhaps reword this to 'get log entry', surely there wouldn't be a plural here as no two log entries would have the same ID?